doc/install.txt

```diff
@@ -412,7 +412,7 @@ that we use.
 3) Install the ATLAS library. ATLAS is an open source optimized version of
 BLAS. You can install a precompiled version on most OSes, but if you're willing
 to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especialy on more recent computers, against the
+speed-ups of up to 3×, especially on more recent computers, against the
```

Comment:
@abergeron The accent problem was due to the fact that setup.py reads from NEWS.txt to fill the LONG_DESCRIPTION field of the package, and it did not specify an encoding.
Now it uses utf-8, and I think the problem is fixed.